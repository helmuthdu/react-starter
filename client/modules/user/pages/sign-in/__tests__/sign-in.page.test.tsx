import { render } from '@testing-library/react';
import React from 'react';
import SignInPage from '../sign-in.page';

describe('user/page -> SignIn component', () => {
  const props: any = {
    name: 'john doe'
  };

  it('should match snapshot', () => {
    const { asFragment } = render(<SignInPage {...props} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
