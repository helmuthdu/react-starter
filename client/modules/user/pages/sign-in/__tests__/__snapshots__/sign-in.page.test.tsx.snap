// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user/page -> SignIn component should match snapshot 1`] = `
<DocumentFragment>
  <form>
    <input
      name="email"
      type="email"
    />
    <input
      name="password"
      type="password"
    />
    <button
      type="submit"
    >
      Submit
    </button>
  </form>
</DocumentFragment>
`;
